### DISCUSSION TITLE
Github marks React as JavaScript language

### DISCUSSION THREAD

**Discussion by usmanawans** [community/community#81299] [Repositories]
Github has marked [React repositoty](https://github.com/facebook/react) as JavaScript language in the sidebar.

<img src="https://github.com/community/community/assets/10607741/3637d332-fe93-475a-acec-2fff287aff6f" width="300" height="147">

**Question:** Is it due to the fact that React development is actually a JavaScript development in foundations?

Kind regards
Usman Awan

**Comment by ananuness** [comment]
Hii! Answering your question... Yes, it is because react is basically JavaScript. Every lib or framework that you use will show the main language used! 

> Also, if you use typescript in your react app, it will show typescript instead! 

For example, a Vue, Angular or Svelte application will be marked as JavaScript, a flutter project will be marked as Dart and so on! I hope I've helped 💗
